web-components/src/components/modal/wallet-modal/components/WalletModal.Mobile.config.tsx

```diff
@@ -0,0 +1,43 @@
+import QRCode from 'qrcode.react';
+
+import { Center } from '../../../../components/box';
+import ContainedButton from '../../../../components/buttons/ContainedButton';
+import { IconTabProps } from '../../../../components/tabs/IconTab';
+import { Subtitle } from '../../../../components/typography';
+
+type getWalletModalMobileTabsParams = {
+  mobileConnectUrl?: string;
+  uri?: string;
+};
+
+export const getWalletModalMobileTabs = ({
+  mobileConnectUrl,
+  uri,
+}: getWalletModalMobileTabsParams): IconTabProps[] =>
+  [
+    mobileConnectUrl
+      ? {
```

Comment:
similar thing here to another comment i added. i'd be grateful to have this without the ternary, if possible/you don't mind.

Reply:
I'm confused, isn't it the same file that you commented on here: https://github.com/regen-network/regen-web/pull/1514#discussion_r1007304128 ?

Reply:
whoops! sorry about that, i got lost in the code somehow 😆